src/nnfusion/frontend/onnx_import/onnx.cpp

```diff
@@ -51,7 +51,45 @@ namespace nnfusion
             load_onnx_model(const std::string& path,
                             const std::unordered_map<std::string, size_t>& dim_params)
         {
-            std::ifstream ifs{path, std::ios::in | std::ios::binary};
+            NNFUSION_LOG(INFO) << "Optimizing ONNX Graph with External Tool "
+                                  "(models/pytorch2onnx/ort_run_frozen.py)";
+            string optimized_filename = string(tmpnam(nullptr));
+            string m_path = path;
+            string script_path =
+                nnfusion::codegen::get_file_from_templates("onnx/ort_run_frozen.py");
+            string cmd = "python3 " + script_path +
+                         " --graph_optimization_level ORT_ENABLE_BASIC "
+                         "--warmup 1 --iters 0 --provider CPUExecutionProvider --file " +
+                         path + " --optimized_model_filepath " + optimized_filename;
```

Comment:
clean optimized file when exit?

Reply:
done

Reply:
Just check optimized_filename existence instead of opt_success flag? Because in some cases, the optimizied model may be sucessfully dumped, but ort_run_frozen script fails (e.g. OOM when do inference)

Reply:
sure, updated